packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemWithMenu.tsx

```diff
@@ -1,70 +0,0 @@
-import * as React from 'react';
-import { BaseComponent, css } from '../../../../Utilities';
-import { IPeoplePickerItemWithMenuProps } from './PeoplePickerItem.types';
-import { Persona, PersonaPresence } from '../../../../Persona';
-import { ContextualMenu, DirectionalHint } from '../../../../ContextualMenu';
-import { IconButton } from '../../../../Button';
-import { FocusZone } from '../../../../FocusZone';
-
-import * as stylesImport from './PickerItemsDefault.scss';
-const styles: any = stylesImport;
-
-/** PeoplePickerItemWithMenu state interface. */
-export interface IPeoplePickerItemWithMenuState {
-  contextualMenuVisible: boolean;
-}
-
-/**
- * PeoplePickerItem with an additional contextual menu.
- * @deprecated Do not use. Will be removed in Fabric 7.0
- */
-// tslint:disable-next-line:deprecation
-export class SelectedItemWithMenu extends BaseComponent<IPeoplePickerItemWithMenuProps, IPeoplePickerItemWithMenuState> {
```

Comment:
Is it ok to just remove this?

Reply:
the comment says:  `@deprecated Do not use. Will be removed in Fabric 7.0`. 
so i assumed so

Reply:
I mean, we did add that comment, but we didn't remove it in 7 so I wonder if we should wait till 8. We have things that say "to be removed in Fabric 5" that are still there.

Reply:
sure i can revert just to be safe

Reply:
`SelectedItemWithMenu` is not a publicly facing component (not exported in our index.js).
and it's not used anywhere internally. i think we are good to remove!